environment/config.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+requirejs.config({
+    baseUrl: '/',
+    paths: {
+        css: '/node_modules/require-css/css',
+        json: '/node_modules/requirejs-plugins/src/json',
+        text: '/node_modules/text/text',
+
+        /* TEST related */
+        'qunit-parameterize': '/environment/qunit2-parameterize',
+        'qunit-assert-close': '/node_modules/qunit-assert-close/qunit-assert-close',
+        qunit: '/node_modules/qunit/qunit/qunit',
+        qunitStyle: '/node_modules/qunit/qunit/qunit',
+        'taoQtiItem/test': '/test',
+        'test/taoQtiItem': '/test',
```

Comment:
It looks really strange. I think we should use one direction.

Reply:
`'taoQtiItem/test'` matches what we already had. I don't think the other is used anywhere.